src/lib/stitching/causality/stitching.ts

```diff
@@ -118,16 +118,47 @@ export const causalityStitchingEnvironment = ({
           // (eg first, after, last, before) are forwarded automatically, so we only
           // need the userId.
           resolve: (parent, _args, context, info) => {
-            return info.mergeInfo.delegateToSchema({
-              schema: causalitySchema,
-              operation: "query",
-              fieldName: "_unused_auctionsLotStandingConnection",
-              args: {
-                userId: parent.internalID,
-              },
-              context,
-              info,
-            })
+            return info.mergeInfo
+              .delegateToSchema({
+                schema: causalitySchema,
+                operation: "query",
+                fieldName: "_unused_auctionsLotStandingConnection",
+                args: {
+                  userId: parent.internalID,
+                },
+                context,
+                info,
+              })
+              .then(async (lotStandingsConnection) => {
+                const promisedSaleArtworks = lotStandingsConnection.edges.map(
+                  ({ node: { lot } }) => {
```

Comment:
One general graphql question: When graphql stitches `_unused_auctionsLotStandingConnection` does it load all the properties? Or does it lazy load things that are being used in the resolver? or it only fetches the fields that the query has asked for?

So in this case specifically is there a guarantee that the `edges->node->lot->internalID` is available even when it is not in the original query? Or we need to add a fragment similar to [this](https://github.com/artsy/metaphysics/pull/2885/files#diff-2a0633fb36f80b8a6471189fc8867efc5686541af5a1be81d4d880932f1edb9aR111) to make sure it is loaded?

Reply:
I will trace the code and verify 👆🏽

Reply:
This actually the case 😬 without `internalID` under `lot` in the query the resolver doesn't return anything. I will try to see if it is possible to add something in the fragment there to make make sure that is always being fetched...